### PR TITLE
Gui mappings

### PR DESCRIPTION
**Pull request by shedaniel**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/widget/VideoSettingsListWidget.mapping [hunk 1]
```diff
@@ -1,3 +1,6 @@
 CLASS cry net/minecraft/client/gui/widget/VideoSettingsListWidget
+	CLASS cry$a VideoSettingsRowEntry
+		FIELD d btnLeftOrFull Lcrm;
```
Comment (shedaniel) [inline mappings/net/minecraft/client/gui/widget/VideoSettingsListWidget.mapping:3]: I am not sure about this line

File: mappings/net/minecraft/client/gui/hud/spectator/SpectatorTeleportMenu.mapping [hunk 2]
```diff
@@ -1,2 +1,5 @@
 CLASS cyr net/minecraft/client/gui/hud/spectator/SpectatorTeleportMenu
+	FIELD a entryOrdering Lcom/google/common/collect/Ordering;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/hud/spectator/SpectatorTeleportMenu.mapping:2]: Not static final?

File: mappings/net/minecraft/client/gui/menu/AddServerScreen.mapping [hunk 3]
```diff
@@ -1,2 +1,9 @@
 CLASS cuf net/minecraft/client/gui/menu/AddServerScreen
+	FIELD a btnAdd Lcrm;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/AddServerScreen.mapping:2]: buttonAdd

File: mappings/net/minecraft/client/gui/menu/AddServerScreen.mapping [hunk 4]
```diff
@@ -1,2 +1,9 @@
...
 	FIELD f parent Lcut;
+	FIELD g serverEntry Ldcw;
+	FIELD h fieldAddress Lcrq;
+	FIELD i fieldServerName Lcrq;
+	FIELD s btnResourcePackState Lcrm;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/AddServerScreen.mapping:7]: buttonResourcePackState

File: mappings/net/minecraft/client/gui/menu/DirectConnectServerScreen.mapping [hunk 5]
```diff
@@ -1 +1,5 @@
 CLASS cud net/minecraft/client/gui/menu/DirectConnectServerScreen
+	FIELD a btnSelectServer Lcrm;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/DirectConnectServerScreen.mapping:2]: buttonSelectServer

File: mappings/net/minecraft/client/gui/menu/settings/LanguageSettingsScreen.mapping [hunk 6]
```diff
@@ -1,6 +1,11 @@
 CLASS cuj net/minecraft/client/gui/menu/settings/LanguageSettingsScreen
+	CLASS cuj$a LanguageSelectionListWidget
+	FIELD a parent Lcut;
+	FIELD f languageSelectionList Lcuj$a;
 	FIELD g settings Lcqp;
 	FIELD h languageManager Ldqi;
+	FIELD i btnForceUnicode Lcrx;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/settings/LanguageSettingsScreen.mapping:7]: buttonForceUnicode

File: mappings/net/minecraft/client/gui/menu/settings/LanguageSettingsScreen.mapping [hunk 7]
```diff
@@ -1,6 +1,11 @@
...
+	FIELD s btnDone Lcrx;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/settings/LanguageSettingsScreen.mapping:8]: buttonDone

File: mappings/net/minecraft/client/gui/menu/settings/SnooperSettingsScreen.mapping [hunk 8]
```diff
@@ -1,5 +1,8 @@
 CLASS cuw net/minecraft/client/gui/menu/settings/SnooperSettingsScreen
+	CLASS cuw$a SnooperInfoListWidget
 	FIELD a parent Lcut;
 	FIELD f settings Lcqp;
+	FIELD t snooperInfoList Lcuw$a;
+	FIELD u btnAllowSnooper Lcrm;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/settings/SnooperSettingsScreen.mapping:6]: buttonAllowSnooper

File: mappings/net/minecraft/client/gui/widget/LabelWidget.mapping [hunk 9]
```diff
@@ -1,4 +1,12 @@
 CLASS crs net/minecraft/client/gui/widget/LabelWidget
+	FIELD a width I
+	FIELD g x I
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/widget/LabelWidget.mapping:3]: x and no y?

File: mappings/net/minecraft/client/gui/widget/OptionSliderWidget.mapping [hunk 10]
```diff
@@ -1 +1,3 @@
 CLASS crk net/minecraft/client/gui/widget/OptionSliderWidget
+	FIELD a client Lcqn;
+	FIELD f slidAmount D
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/widget/OptionSliderWidget.mapping:3]: slidAmount? not position or something?

**Review by asiekierka — CHANGES_REQUESTED** [review]
I think I'm just gonna re-do the PR myself

**Comment by asiekierka** [comment]
https://github.com/FabricMC/yarn/pull/469

**Pull request closed without merging**